lib/screens/employees/create_employee_view.dart

```diff
@@ -462,7 +462,7 @@ class _CreateEployeeViewState extends State<CreateEployeeView> {
       maxRadius: 18.0,
       foregroundImage: CachedNetworkImageProvider(selectedEmployee.avatarAttachmentUrl ?? ""),
       child: Text(
-        getInitials(selectedEmployee.name ?? "", 2),
+        getInitials(selectedEmployee.name ?? "", 2).toUpperCase(),
```

Comment:
@sriLatha2328  toUpperCase function to be moved inside the getInitials function so that it handles the upper cases wherever used.